pages/common/git-config.md

```diff
@@ -1,27 +1,28 @@
 # git config
 
-> Get and set repository or global options.
+> Manage custom configuration options for git repositories.
+> These configurations can be local (for the current repository) or global (for the current user).
 
-- Print list of options for current repository:
+- List only local configurations (stored in `.git/config` in the current repository):
 
 `git config --list --local`
 
-- Print global list of options, set in ~/.gitconfig:
+- List only global configurations (stored in `~/.gitconfig`):
 
 `git config --list --global`
 
-- Get full list of options:
+- List all configurations that have been defined either locally or globally:
```

Comment:
> You are picking from a list of configuration variables though - it's just the values that are different

Not necessarily -- as I mention above, the entries are in the form `<section>.<key> = <value>`, and for most of them only the value is user-provided (e.g. `color.ui = true` or `merge.tool = meld`), but for aliases, the actual key is user-provided as well (e.g. `alias.st = status`).